VideoRecStation/src/speakerthread.cpp

```diff
@@ -26,36 +26,36 @@ using namespace std;
 
 SpeakerThread::SpeakerThread(NonBlockingBuffer* _buffer)
 {
-	int						rc;
-	snd_pcm_hw_params_t*	params;
-	unsigned int			sampRate;
-	snd_pcm_uframes_t		framesPerPeriod;
+    int                     rc;
+    snd_pcm_hw_params_t*    params;
+    unsigned int            sampRate;
+    snd_pcm_uframes_t       framesPerPeriod;
 
-	buffer = _buffer;
+    buffer = _buffer;
 
-	// Open PCM device for playback
-	rc = snd_pcm_open(&sndHandle, settings.outAudioDev, SND_PCM_STREAM_PLAYBACK, 0);
-	if (rc < 0)
-	{
-		cerr << "unable to open pcm device: " << snd_strerror(rc) << endl;
-	    exit(EXIT_FAILURE);
-	}
+    // Open PCM device for playback
+    rc = snd_pcm_open(&sndHandle, settings.outAudioDev.toLocal8Bit().data(), SND_PCM_STREAM_PLAYBACK, 0);
```

Comment:
`settings.outAudioDev`, `settings.inAudioDev`, and `settings.storagepath` were all replaced with QString instances, so I now change them over to the local8Bit when necessary.